lib/decompress/zstd_decompress_block.c

```diff
@@ -1170,9 +1170,19 @@ FORCE_INLINE_TEMPLATE seq_t
 ZSTD_decodeSequence(seqState_t* seqState, const ZSTD_longOffset_e longOffsets)
 {
     seq_t seq;
+    #if defined(__aarch64__) && (defined(__GNUC__) && !defined(__clang__))
+    ZSTD_seqSymbol llDInfoS, mlDInfoS, ofDInfoS;
```

Comment:
Maintenance : 
Please explain in code comment why this code block exists.

Something equivalent to your explanation text in the PR will be good enough.

The intention is for future maintainers to understand why this code block exist.

Reply:
Good suggestion. Will update. Thanks.

Reply:
Updated.